errors/errors.js

```diff
@@ -0,0 +1,30 @@
+const error404 = ('/*', (req, res) => {
+    res.status(404).send({ message: 'Path not found.'});
+  });
+  
+  // ---- custom errors ----
+  const customError = ((err, req, res, next) => {
```

Comment:
Not hurting anyone - but you don't need to double wrap the parentheses around the next 3 handlers

Reply:
done!